lib/pyspark_preprocesing.py

```diff
@@ -0,0 +1,33 @@
+from pyspark.ml.feature import StringIndexer, OneHotEncoder
+from pyspark.sql import DataFrame
+from typing import List
+
+
+def one_hot_encoding(
```

Comment:
to się prosi o unit-test

Reply:
napisałem test w tests/test_one_hot_encoding.py